plugins/flytekit-pandera/flytekitplugins/pandera/schema.py

```diff
@@ -95,7 +95,7 @@ def downloader(x, y):
             downloader=downloader,
             supported_mode=SchemaOpenMode.READ,
         )
-        return self._pandera_schema(expected_python_type)(df.open().all())
+        return pandera.typing.DataFrame(self._pandera_schema(expected_python_type)(df.open().all()))
```

Comment:
On the topic of the type change, I'm not really sure what the desired behavior would be.

1. On one hand, we want the output of this to be of the specific pandera dataframe type `pandera.typing.DataFrame[<SCHEMA>]`. If this is the desired behavior, this should be `expected_python_type(self._pandera_schema(...))`, since `expected_python_type` is the generic type with the `<SCHEMA>` argument.
2. On the other, I'm not sure what the implications would be if we change the return signature of `to_python_value` to `pandas.DataFrame`... I guess this would violate a user expectation that the data being passed into the task function body should be.

So I'd opt for option (1), assuming that `expected_python_type(self._pandera_schema(...))` works.